PerimeterXModule/DataContracts/Activities/ActivityDetails.cs

```diff
@@ -16,6 +16,11 @@ public class ActivityDetails
 
 		[DataMember(Name = "block_score")]
 		public double RiskScore;
-	}
 
+		[DataMember(Name = "pass_reason", EmitDefaultValue = true)]
+		public PassReasonEnum? PassReason;
+
+		[DataMember(Name = "risk_rtt")]
+		public double RiskRoundtripTime;
```

Comment:
You eventually assign a `long` value to this property (`Stopwatch.ElapsedMilliseconds`), so save the implicit casting and declare it as `long` in the first place.

While I'm nitpicking your type decisions, I see that `RiskScore` is also `double`, when  it should actually be `int` (even `short` or `byte` will do for a number between 0-100).